packages/react-charting/docs/TestPlans/VerticalBarChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+|                                   Test 8: [Sub-Component]: Line                                   |                                                                                                                                        |               |
+|                                        - Specify line data                                        |                                               Should render line with the data provided                                                |      RTL      |
+|                                - Hover mouse over the data points                                 | Should highlight the data points (No callout is rendered when we hover only on the line. Callout appears on hover over the bars only.) |      RTL      |
+|                                 Test 9: [Sub-Component]: Legends                                  |                                                                                                                                        |               |
+|                                          - Hide legends                                           |                                                  Should not show any rendered legends                                                  |      RTL      |
+|                                  - Hover mouse over bar legends                                   |                                  Should reduce the opacity of the other bars/lines and their legends                                   |      RTL      |
+|                                  - Hover mouse over line legends                                  |                                  Should reduce the opacity of the other bars/lines and their legends                                   |      RTL      |
+|                                 Test 10: [Sub-Component]: Callout                                 |                                                                                                                                        |               |
+|                                     - Hover mouse over a bar                                      |                                               Should call the handler on mouse over bar                                                |      RTL      |
+|                                     - Hover mouse over a bar                                      |                                             Should show the default callout over that bar                                              |      RTL      |
+|                                    - Hover mouse over the line                                    |                                             Should show the default callout over that line                                             |      RTL      |
```

Comment:
Hovering mouse over line does not show the callout. The callout is shown only if mouse is hovered over a bar.

Reply:
yes, mentioned in the above validation step (Test case 8b) but did not remove this test case.

Reply:
removed now